### PR TITLE
Create 2020-01-08-11.000users.md

### PR DESCRIPTION
**Pull request by erxleben**

### CODE REVIEW

**Comment by bgruening** [comment]
Should we do this every 1000? maybe every 5000 is enough? Not sure, but currently we are getting ~1000 every month.

**Comment by erxleben** [comment]
Mhm ... okay, then in February the recent user number (maby 12 or 13K) for the de.NBI reviewers.

**Comment by bgruening** [comment]
Let's play a little bit with number: 250.000 jobs a month, means 8300 jobs per day, 350 jobs per hour, 6 per minute or every 11 seconds verliebt sich ein Single auf ... äh ich meine 11 seconds finishes one job in Galaxy.

Every active user executes 166 jobs on average a month.

File: _posts/2020-02-01-12.000users.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+site: freiburg
+tags: [devops]
+title: 12,000 users - 6,900,000 jobs - 13,300,000 datasets
+supporters:
+- denbi
+- elixir
+---
+
+2020 continous to be successful! There are **12,000 Galaxy users** on https://usegalaxy.eu; you run 6,900,000 jobs on our European Galaxy server; and you have created 13,300,000 datasets!
```
Comment (simonbray) [inline _posts/2020-02-01-12.000users.md]:
```suggestion
2020 continues to be a successful year! There are now **12,000 Galaxy users** on https://usegalaxy.eu; you have run 6,900,000 jobs on our European Galaxy server, and created 13,300,000 datasets! 
```

File: _posts/2020-02-01-12.000users.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+supporters:
+- denbi
+- elixir
+---
+
+2020 continous to be successful! There are **12,000 Galaxy users** on https://usegalaxy.eu; you run 6,900,000 jobs on our European Galaxy server; and you have created 13,300,000 datasets! 
+
+This success would not be possible without the support of [de.NBI](https://www.denbi.de/), [ELIXIR](http://elixir-europe.org/), [EOSC-life](https://www.eosc-portal.eu/eosc-life) and the Freiburg [Faculty of Medicine](http://www.med.uni-freiburg.de).
+
+A big thank you to all our users and people that have contributed to this project! Special thanks to our partner sites [Erasmus MC](https://galaxyproject.eu/erasmusmc/), [Belgium](https://galaxyproject.eu/vib/), and [Pasteur](https://galaxyproject.eu/pasteur/) and the whole [Galaxy community](https://galaxyproject.org/community/).
```
Comment (simonbray) [inline _posts/2020-02-01-12.000users.md]:
```suggestion
A big thank you to all our users and to everyone that have contributed to this project! Special thanks to our partner sites [Erasmus MC](https://galaxyproject.eu/erasmusmc/), [Belgium](https://galaxyproject.eu/vib/), and [Pasteur](https://galaxyproject.eu/pasteur/) and the whole [Galaxy community](https://galaxyproject.org/community/).
```

**Comment by bgruening** [comment]
> 6 per minute or every 11 seconds verliebt sich ein Single auf

@stephanflemming you probably know some meme that we can use here? To create a catchy image?

**Comment by stephanflemming** [comment]
How comes that you know this phrase? Are you watching german trash TV? :-) I will think about the meme.

**Comment by bgruening** [comment]
@stephanflemming https://youtu.be/JWs2xiyYY2U?t=275

**Pull request merged**